modules/virt-creating-bridge-nad-cli.adoc

```diff
@@ -53,10 +59,10 @@ $ oc create -f <network-attachment-definition.yaml> <1>
 
 .Verification
 
-* Verify that the network attachment definition is created by running either `oc get network-attachment-definition <name>` or `oc get net-attach-def <name>`. For example:
+* Verify that the network attachment definition was created:
+** Run `oc get network-attachment-definition <name>`. For example:
```

Comment:
Not critical, but this seems repetitive to have this line in here with the command, when the command is shown below that still uses a replaceable value.

I'd consider removing this line and just modify the first sentence to say something like `Verify that the network attachment definition was created by running the following command:`

Reply:
Agreed.